src/Tools/IiifHelper.php

```diff
@@ -13,46 +13,50 @@
 class IiifHelper {
 
   use StringTranslationTrait;
-  
-  protected $remoteInfoJsonURL;
 
+  protected $publicUrl;
+  protected $identifier;
+  protected $internalUrl;
 
   /**
    * @var array;
    */
-  protected $remoteInfoJson;
+  protected $remoteInfoJsonData;
```

Comment:
Don't rename methods if not really really needed. We are not working alone and someone else could have coded against that... but then this one is protected.. so not that terrible... So why the extra data?